### PR TITLE
More Japanese

### PR DESCRIPTION
**Pull request by LuminaryXion**
Trying to add an additional update. Having issues with the process. Hopefully I got it this time!

### CODE REVIEW

**Comment by PieterGit** [comment]
@LuminaryXion do you want to be a maintainer for the Japanese language, if so please PR https://github.com/nightscout/cgm-remote-monitor/blob/dev/CONTRIBUTING.md#translators

**Comment by LuminaryXion** [comment]
Hi! I've added myself as a Japanese contributor. Hopefully I got it right. :)

**Comment by LuminaryXion** [comment]
resolved the conflicts. :)

**Comment by LuminaryXion** [comment]
resolved the conflicts. Should be good to go now!

**Comment by LuminaryXion** [comment]
@PieterGit Hi Pieter! Is this mergable yet? Thank you!

**Comment by PieterGit** [comment]
@LuminaryXion You changed some whitespacing, which does not look good on first sight. Please make sure https://github.com/nightscout/cgm-remote-monitor/pull/4150/files only lists japanse fixes and don't make changes to the whitespacing

**Comment by LuminaryXion** [comment]
Hi @PieterGit I had changed the whitespace to be consistent, since it got strange and uneven at the end of the file.

I don't see where it looks odd, could you specify an example and I'll check it out? Thanks!

File: lib/language.js [hunk 1]
```diff
@@ -12573,913 +12667,912 @@ function init() {
...
+	,ro: 'Aveți %1 insulină activă'
+	,ru: 'вы имеете %1 инсулина в организме'
+	,sk: 'You have %1 insulin on board'
+	,sv: 'You have %1 insulin on board'
+	,nl: 'You have %1 insulin on board'
+	,tr: 'Sizde %1 aktif insülin var'
+	,zh_cn: '你有 %1 的活性胰岛素'
+	,zh_tw: 'You have %1 insulin on board'
     },
-     'alexaIobIntent': {
```
Comment (PieterGit) [inline lib/language.js]: why did you remove alexaIobIntent?

**Comment by LuminaryXion** [comment]
Ok, I found an easy-ish way to manually put it back and did so. Looks all good to me now. Sorry for the trouble! My OCD kicked in and tried to help. xD

LMK if anything else. :)

**Comment by PieterGit** [comment]
LGTM. Merging to `dev`.
